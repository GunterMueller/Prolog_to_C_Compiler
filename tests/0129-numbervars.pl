main :-
	T = foo(X, Y, [X|Z], yes, no(Z)),
	numbervars(T, 0, _),
	writeq(T), nl.
