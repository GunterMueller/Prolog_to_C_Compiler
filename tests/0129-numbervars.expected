foo('$VAR'(0),'$VAR'(1),['$VAR'(0)|'$VAR'(2)],yes,no('$VAR'(2)))
